src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -379,4 +379,12 @@ public function getPaginationLimitChoices()
 
         return $paginationLimitChoices;
     }
+
+    /* (non-PHPdoc)
```

Comment:
Because the next line is @see \PrestaShopBundle\Service\DataProvider\Admin\ProductInterface::isNewProductDefaultActivated()
and indicates to use the parent's PHPDoc... the standard I use from the begining.